src/common/IOSEnvironmentRequirements.ts

```diff
@@ -9,17 +9,32 @@ import util from 'util';
 import { CommonUtils } from './CommonUtils';
 import { IOSUtils } from './IOSUtils';
 import { PlatformConfig } from './PlatformConfig';
-import { Requirement, RequirementList } from './Requirements';
+import {
+    CommandRequirements,
+    Requirement,
+    RequirementList
+} from './Requirements';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages(
     '@salesforce/lwc-dev-mobile-core',
     'requirement-ios'
 );
 
-export class IOSEnvironmentRequirements implements RequirementList {
+export class IOSEnvironmentRequirements {
+    public commandRequirements: CommandRequirements;
+
+    constructor(logger: Logger) {
+        this.commandRequirements = {};
+
+        this.commandRequirements.r1 = new IOSRequirements(logger);
```

Comment:
Doesn't have to be done in this PR if you don't want scope creep, but it would be nice to break this up similar to the Android counterpart.